web/modules/custom/dpl_example_content/dpl_example_content.info.yml

```diff
@@ -29,6 +35,9 @@ default_content:
     - dc33677f-8894-4717-b34b-d985f6ad875f
     # embed video in article 1
     - f0204f0b-1f63-4c76-bb67-e3f7489ee392
+    # Bibliotekarerne anbefaler læsning til den mørke tid
+    - 7b9d1894-7f9a-4f96-a7dc-efa3af5fc902
+    - a6a77786-07db-48d8-98ff-3bda54f231bf
```

Comment:
Please reconsider this comment.

As I read it, these two media items are specifically related to the article "Bibliotekarerne anbefaler læsning til den mørke tid" but when I check the dependencies for that item it uses 5 different media entities.

Either we should update our commenting to show that a media item is used multiple times or we should drop the commenting because it quickly becomes unmanagable.